handbook/engineering/README.md

```diff
@@ -141,7 +132,7 @@ This would include the CTO + Engineering managers.
 
 This section outlines the release process at Fleet.
 
-The current release cadence is once every three weeks and is concentrated around Wednesdays.
+The current release cadence is nightly.
```

Comment:
By my understanding, we now have nightly releases as-needed (when there is something ready to ship) for the Fleet Sandbox. The normal Fleet releases continue to be every three weeks.

Reply:
Ok, so then we'll just leave this line as is and change the other line to reflect the goals.